src/index.html

```diff
@@ -11,8 +11,65 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.png"
+        alt="imac image"
+        class="card__image"
+      />
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <h4 class="card__code">Product code: 195434</h4>
+
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card__price-block">
+        <h3 class="card__price-title">Price:</h3>
+
+        <h1 class="card__price-amount">$2,199</h1>
+      </div>
+
+      <button class="card__buy-button">
+        <a
+          data-qa="hover"
+          href="#"
+          class="card__buy-button-link"
+        >
+          Buy
+        </a>
+      </button>
```

Comment:
There's a semantic issue here. The `<button>` element should not contain an `<a>` link. Instead, you should use either a `<button>` or an `<a>`, but not both. If the intention is to navigate, use the `<a>` tag with button styles. If the intention is to submit a form or trigger a JavaScript action, use the `<button>` tag.